src/style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  max-width: 200px;
+  padding: 14px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product-card__image {
+  margin: 15px 6px 35px;
+  width: 160px;
+  height: 134px;
+}
+
+.product-card__description {
+  width: 100%;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 15px;
+  text-align: left;
```

Comment:
```suggestion
```
You can remove this if you will remove 'text-align: center;' from your `.product-card`